.travis.yml

```diff
@@ -4,10 +4,11 @@ python:
   - "3.5"
   - "3.6"
   - "3.7"
+  - "3.12"
```

Comment:
Can you make the test suite test:
- Python 3.7 (deb 10)
- Python 3.9 (deb 11)
- Python 3.11 (deb 12)

Think the others like python < 3.7 and python 3.12 may be a bit overkill for now.